providers/os/resources/python.go

```diff
@@ -210,8 +208,9 @@ func collectPythonPackages(runtime *plugin.Runtime, fs afero.Fs, path string) ([
 
 	fileList, err := afs.ReadDir(path)
 	if err != nil {
-		if !os.IsNotExist(err) {
+		if os.IsNotExist(err) {
```

Comment:
the ! felt wrong to me, i think this should be just `if os.IsNotExist` instead of `if !os...`